packages/blockchain-wallet-v4-frontend/src/scenes/Settings/Addresses/Bch/Wallets/template.success.js

```diff
@@ -160,7 +161,16 @@ const Success = props => {
                     defaultMessage='Show xPub'
                   />
                 </ClickableText>
-              )
+              ),
+              <ClickableText
```

Comment:
I think we should use selector to determine the difference between btc and bch change addresses and only show link in this case